tools/aplcms/aplcms_macros.xml

```diff
@@ -1,9 +1,9 @@
 <macros>
-    <token name="@TOOL_VERSION@">6.6.6</token>
+    <token name="@TOOL_VERSION@">6.6.8-recetox0</token>
 
     <xml name="requirements">
         <requirements>
-            <container type="docker">recetox/aplcms:6.6.6-recetox2</container>
+            <requirement type="package" version="0">r-recetox-aplcms</requirement>

```

Comment:
```suggestion
            <requirement type="package" version="0">r-recetox-aplcms</requirement>
            <requirement type="package" version="1.3.2">r-hdf5r</requirement>

```

Reply:
Also add HDF5 dependency since it's not present in r-recetox-aplcms package.

Reply:
Okay, this may be unnecessary due to https://github.com/RECETOX/recetox-aplcms/issues/13